### ISSUE TITLE
Run tests inside their project directory

### ISSUE THREAD

**Issue by roji** [JetBrains/teamcity-dotnet-plugin#18]
The plugin in its current state runs tests from outside the project directory - this can cause issues with tests which access file resources, assuming that the current directory is the project.

In other words, rather than doing `dotnet test test/myproj`, it would be better to do `cd test/myproj; dotnet test`.

**Comment by mcctomsk** [comment]
We've got error message after we switched from command line step
----
cd FolderWithTest
dotnet test
----
to using step with plugin.
[TestName] System.AggregateException : One or more errors occurred. (The configuration file 'appsettings.json' was not found and is not optional.) (The configuration file 'appsettings.json' was not found and is not optional.) (The following constructor parameters did not have matching fixture data: ServerFixture fixture)
---- System.IO.FileNotFoundException : The configuration file 'appsettings.json' was not found and is not optional.
---- System.IO.FileNotFoundException : The configuration file 'appsettings.json' was not found and is not optional.
---- The following constructor parameters did not have matching fixture data: ServerFixture fixture
